src/panels/config/helpers/forms/ha-schedule-form.ts

```diff
@@ -389,12 +427,45 @@ class HaScheduleForm extends LitElement {
           -ms-user-select: none;
           user-select: none;
         }
-        .fc-scroller {
-          overflow-x: visible !important;
-        }
+
         .fc-v-event .fc-event-time {
           white-space: inherit;
         }
+        .fc-theme-standard .fc-scrollgrid {
+          border: 1px solid var(--divider-color);
+          border-radius: 4px;
```

Comment:
```suggestion
          border-radius: var(--ha-card-border-radius, 12px);
```

Reply:
See decision in https://github.com/home-assistant/frontend/pull/14525